x/gamm/client/cli/cli_test.go

```diff
@@ -921,6 +921,43 @@ func (s *IntegrationTestSuite) TestGetCmdTotalShare() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestGetCmdTotalLiquidity() {
+	val := s.network.Validators[0]
+
+	testCases := []struct {
+		name      string
+		args      []string
+		expectErr bool
+	}{
```

Comment:
Can we create a pool and add a test that the total liquidity is correct, or is that too inconvenient to do for the CLI tests?

Reply:
I think it's complex to do in CLI as CLI has lots of other tests doing in parallel.